fhir-server-test/src/test/java/testng.xml

```diff
@@ -58,4 +58,10 @@
             <package name="com.ibm.fhir.server.test.performance" />
         </packages>
     </test>
-</suite>
\ No newline at end of file
+    <test name="TerminologyTests">
+        <packages>
+            <!-- All of these terminology tests are disabled by default, and enabled in test.properties -->
```

Comment:
is this actually the case?  I don't think it can be disabled.

Reply:
good catch.  originally corey wrote it so that it can be enabled/disabled from test.properties.  I requested him to remove that since the terminology stuff gets baked into the server...there should be no need to disable them in this way
```suggestion
            <!-- All of these tests are automatically executed -->
```